src/services/userService.js

```diff
@@ -0,0 +1,59 @@
+const userDao = require('../models/userDao')
+const jwt = require('jsonwebtoken')
+const axios = require('axios');
+
+const kakaoLogin = async (authCode) => {
+    try { 
+        const getToken = await axios.get('https://kauth.kakao.com/oauth/token', {
+            headers: {
+                'Content-Type': 'application/x-www-form-urlencoded'
+            },
+            params: {
+                grant_type: 'authorization_code',
+                client_id: process.env.REST_API_KEY,
+                redirect_url: process.env.REDIRECT_URI,
+                code: authCode
+            },
+            withCredentials: true
+        })
+
+        const getUserData = await axios.get('https://kapi.kakao.com/v2/user/me', {
+            headers: {
+                'Authorization': `Bearer ${getToken.data.access_token}`
+            }
+        })
+
+        const kakaoId = getUserData.data.id
+        const email = getUserData.data.kakao_account.email
+        const profileImage = getUserData.data.properties.profile_image
+        const nickname = getUserData.data.properties.nickname
```

Comment:
```suggestion
        const {profile_image : profileImage, nickname } = getUserData.data.properties
```
두 데이터 모두 같은 depth에 위치하고 있기 때문에 다음과 같이 구조분해할당을 활용하여 코드를 한 줄이라도 줄일 수 있을 것 같네요 :)